test/skimOutputter_HHbtag.cpp

```diff
@@ -1389,6 +1432,152 @@ int main (int argc, char** argv)
     // Timing info
     auto end_prep = high_resolution_clock::now();
 
+    // ---- ---- ---- ---- ---- ---- ---- ----
+    // ---- ----  Do doTauhFake now  ---- ----
+    // ---- ---- ---- ---- ---- ---- ---- ----
+    // Add jet->tau_h SF (always 1) and uncertainty
+    if (doTauhFake)
+    {
+      if (isData)
+      {
+          // No SF for data
+          jetToTauhFakeSF          = 1.0;
+          jetToTauhFakeSF_up       = 1.0;
+          jetToTauhFakeSF_down     = 1.0;
+          jetToTauhFakeSF_bar      = 1.0;
+          jetToTauhFakeSF_bar_up   = 1.0;
+          jetToTauhFakeSF_bar_down = 1.0;
+          jetToTauhFakeSF_end      = 1.0;
+          jetToTauhFakeSF_end_up   = 1.0;
+          jetToTauhFakeSF_end_down = 1.0;
+      }
+      else /*isMC*/
+      {
+        // Logic of jet->tau_h uncertainty:
+        // - the "central" SF is always 1 (i.e. no SF is needed)
+        // - up and down variations are computed depending on the channel (tauTau or muTau/eTau),
+        //   on the position of the taus (barrel or endcap) and on the jetFakeSF branch of the skims
+
+        // Central SF always 1
+        jetToTauhFakeSF     = 1.0;
+        jetToTauhFakeSF_bar = 1.0;
+        jetToTauhFakeSF_end = 1.0;
+
+        // Up/Down variations
+        if (pType == 2) /*tauTau channel*/
+        {
+          // Possible values of "jetFakeSF":
```

Comment:
Maybe I would write in a comment that this is a patch using some obsolete values to infer if the tauh is fake - for future reference

Reply:
yes good point, I will add this comment.

Reply:
done in a5ba973